scripts/generate_sheet.py

```diff
@@ -53,12 +53,18 @@ def generate_sheet(model_name, token):
 
 
 def main():
-    description = "Perform evaluation on bugs from last week on the specified model and generate a csv file "
+    description = "Perform evaluation on bugs from specified days back on the specified model and generate a csv file "
     parser = argparse.ArgumentParser(description=description)
 
     parser.add_argument("model", help="Which model to generate a csv for.")
     parser.add_argument("token", help="Bugzilla token")
+    parser.add_argument(
+        "days", type=int, help="No. of days back from which bugs will be evaluated"
+    )
+    parser.add_argument(
+        "threshold", type=float, help="Confidence threshold for the model"
+    )
```

Comment:
Forgot to tell you before: let's use 7 as default for the days, and 0.7 as default for the threshold (to keep the same behavior as before if the arguments are not passed).